examples/cloud_discovery_service/library_api/c++11/README.md

```diff
@@ -68,24 +68,23 @@ in the Cloud Discovery Service User's Manual.
 
 By default, CMake will generate build files using the most common generator for
 your host platform (e.g., Makefiles on Unix-like systems and Visual Studio
-solution on Windows), \. You can use the following CMake variables to modify the
+Solutions on Windows). You can use the following CMake variables to modify the
 default behavior:
 
--   `-DCMAKE_BUILD_TYPE` -- specifies the build mode. Valid values are Release
-    and Debug. See the [CMake documentation for more details.
+-   `-DCMAKE_BUILD_TYPE` - specifies the build mode. Valid values are `Release`
+    and `Debug`. See the [CMake documentation for more details.
```

Comment:
The period is still not in the right place here, right?

Reply:
(I approved anyway)

Reply:
I missed that part. Done now.